### DISCUSSION TITLE
Lubuntu 19.04 preinstalled disk image! (only console works)

### DISCUSSION THREAD

**Discussion by spetterman66** [copy/v86#896] [Show and tell]
# Summary
This is a disk image of Lubuntu 19.04, the last version of Lubuntu to support the i386 CPU architecture.. Only the console mode works. To open the console mode, type CTRL+ALT+F4 and then login with the username and password `v86`.
# Screenshots
![image](https://github.com/copy/v86/assets/69303239/10b4a409-d56d-4f08-845d-ce183571c0b6)
# Download
Download from [pCloud](https://e.pcloud.link/publink/show?code=XZTH3BZwmCR1aIuHBhruu1TuRDP5FUGxUlyl)

**Comment by ZXMushroom63** [comment]
very cool!
I'm going to try to make an image too. I reckon ubuntu server rather than lubuntu, so that it boots up without needing to press ctrl+alt+f4
